mungegithub/mungers/approvers/owners.go

```diff
@@ -150,6 +151,34 @@ func (o Owners) GetOwnersSet() sets.String {
 	return removeSubdirs(owners.List())
 }
 
+// IntersectSetsCase runs the intersection between to sets.String in a
+// case-insensitive way. It returns the name with the case of "one".
+func IntersectSetsCase(one, other sets.String) sets.String {
+	lower := sets.NewString()
+	for item := range other {
+		lower.Insert(strings.ToLower(item))
+	}
+
+	intersection := sets.NewString()
+	for item := range one {
+		if lower.Has(strings.ToLower(item)) {
+			intersection.Insert(item)
+		}
+	}
+	return intersection
+}
+
+// GetApprovalStatus returns a map from files -> list of current approvers.
+func (o Owners) GetApprovalStatus(approvers sets.String) map[string]sets.String {
+	status := map[string]sets.String{}
```

Comment:
This kind of contradicts our original design of keeping anything tied to the comments separate from the owners struct (which should only be dependent on the PR).  Can we attach this func to Approvers struct 

`func (a Approvers) GetApprovalStatus`